src/core/src/op/paged_attention.cpp

```diff
@@ -146,7 +146,23 @@ void PagedAttentionExtension::validate_and_infer_types() {
                           get_input_element_type(12),
                           ".");
 
-    set_output_type(0, get_input_element_type(0), get_input_partial_shape(0));
+    // value head_size may be not same with key
+    auto out_ps = get_input_partial_shape(0);
+    const auto& key_ps = get_input_partial_shape(1);
+    const auto& value_ps = get_input_partial_shape(2);
+    if (out_ps.rank().is_static()) {
+        out_ps[1] = Dimension::dynamic();
+        if (key_ps.rank().is_static() && value_ps.rank().is_static() && out_ps[1].is_static() &&
+            key_ps[1].is_static() && value_ps[1].is_static()) {
+            // query_ps[1] = num_heads * head_size
```

Comment:
these comments can be removed?

Reply:
The comments are used to explain the following code. The comments are modified to more readable.